@here/harp-datasource-protocol/StyleExpressions.md

```diff
@@ -165,6 +165,33 @@ returns the value of the first `fallback` that is a number.
 ["to-number", value, fallback...]
 ```
 
+## to-vector2
+
+Converts the value to "vector2", if the value cannot be converted to a vector2
```

Comment:
It's not documented what can be successfully converted to `vector*` ?

Reply:
good point, I will write something